client/controllers/incidentModal.coffee

```diff
@@ -22,8 +22,6 @@ Template.incidentModal.events
            $(form.count).val("")
            $(form.incidentType).val(null).trigger("change")
            $(form.travelRelated).attr('checked', false)
-        if closeModal
```

Comment:
No, it looks like a left over from before we changed the modal buttons. You're right to delete it.